styles.css

```diff
@@ -1,3 +1,345 @@
 body {
-  background-color: green;
+  background-color: lightgray;
+}
+
+a {
+  text-decoration: none;
+}
+
+.main-layout {
+  width: 400px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  background-color: white;
+  margin-left: auto;
+  margin-right: auto;
+  overflow: hidden;
+}
+
+nav {
+  height: 50px;
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: white;
+  width: 400px;
+  position: fixed;
+  margin: 0;
+}
+
+nav span {
+  margin: 0 5px;
+}
+
+nav img {
+  margin: 0 5px;
+}
+
+.intro-section {
+  background-color: #1c1a19;
+}
+
+.intro-section h1 {
+  color: #ff6b00;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  background-size: cover;
+  padding: 0 18px;
+}
+
+.intro-section h4 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: #fff5e1;
+  margin: 10px;
+  padding: 0 18px;
+}
+
+.intro-section p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  padding: 0 18px;
+}
+
+.contact-icons a {
+  margin: 20px 7px;
+}
+
+.contact-icons {
+  width: 100%;
+  margin: 0 15px 0 0;
+  height: 100px;
+}
+
+.wazzi {
+  margin-left: 10px;
+}
+
+.bg-handle {
+  background-image: url('./Assets/Group.png');
+  background-repeat: no-repeat;
+  background-position: bottom 300px;
+  background-position: right;
+  background-size: 100px 200px;
+  padding: 0;
+}
+
+.recent-work {
+  width: 80%;
+  font-size: 25px;
+  margin: 15px auto;
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.recent-work h4 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 30px;
+  line-height: 52px;
+  margin: 20px;
+}
+
+.svg-image {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.header {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+  margin-top: 10px;
+  margin-bottom: 0;
+}
+
+.main-layout2 {
+  padding: 18px;
+}
+
+.main-layout2 p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.main-layout2 button {
+  padding: 12px;
+  background: #ff6b00;
+  margin: 7px 0;
+  border: 0 solid transparent;
+  color: white;
+}
+
+.about-section {
+  background: #1c1a19;
+  padding: 18px;
+  display: flex;
+  flex-direction: column;
+  color: #fff;
+}
```

Comment:
- [ ] Please 🙏🏽  correct the padding of the last card,  there is a bigger white space between the works section and the about section on the Figma template.
- [ ] Kindly correct the font style, size, and weight of the title to be similar to the Figma template. 👇🏽 
- Figma template
![image](https://user-images.githubusercontent.com/58642949/187034031-33594c46-b053-4c42-8318-5626b1a3b933.png)

- Your design
![image](https://user-images.githubusercontent.com/58642949/187033994-7e9967c6-1619-4ab7-9246-d8cca3301573.png)